### PR TITLE
Remove support for Python 2

### PR DESCRIPTION
**Pull request by henri-hulski**
This PR removes support for Python 2 as we decided in https://github.com/morepath/reg/issues/55#issuecomment-511098204.

### CODE REVIEW

File: doc/settings.rst [hunk 1]
```diff
@@ -109,7 +109,7 @@ You can load it with:
   import yaml
 
   with open('settings.yml') as config:
-       settings_dict = yaml.load(config)
+       settings_dict = yaml.full_load(config)
```
Comment (href) [inline doc/settings.rst]: Wouldn't safe_load make more sense here?

Comment (henri-hulski) [reply doc/settings.rst]: Probably... I was just not sure, but if you think it's enough, I will change it.
I just don't know, what is the exact difference.

Comment (href) [reply doc/settings.rst]: `full_load` supports unsafe things that could lead to exploits, `safe_load` doesn't. In practice I haven't seen a case yet where `full_load` is necessary. In general it is recommended to always use `safe_load`, so I think we should use that as well.

Comment (henri-hulski) [reply doc/settings.rst]: BTW the FullConstructor is still much safer then the legacy UnsafeConstructor.

Comment (henri-hulski) [reply doc/settings.rst]: Ok I've changed it.

File: CHANGES.txt [hunk 2]
```diff
@@ -4,23 +4,32 @@ CHANGES
 0.19 (unreleased)
 =================
 
-- Nothing changed yet.
+* **Removed**: Removed support for Python 2.
+  
+  You have to upgrade to Python 3 if you want to use this version.
+
+* Added support for Python 3.8 and Pypy 3.6.
```
Comment (faassen) [inline CHANGES.txt]: PyPy, case spelling.

**Comment by henri-hulski** [comment]
Travis is failing because the changes in Reg are not yet in master. Travis uses the master branches of Reg and Dectate.

I wanted to add the dependencies update after Reg and Dectate are released so that the local tests are still working.

File: CHANGES.txt (hunk 2 above)
Comment (henri-hulski) [reply CHANGES.txt]: Fixed here and also in Reg, Dectate and Importscan.

**Comment by henri-hulski** [comment]
@faassen Fixed the dependency versions of reg, dectate and importscan. Local tests are still working.

**Pull request merged**